### ISSUE TITLE
Howto set Footer text size

### ISSUE THREAD

**Issue by yamsu** [MartinPacker/md2pptx#141]
This is a really amazing tool, thanks for creating it!
I have been using this regularly, but have struggled with the template file when it comes to the slide footer
For the following footer, how would one set the text size?
```
leftFooterText: Left Text
middleFooterText: My presentation
rightFooterText: Right Text
```
Is there a way to also set the page number size?

**Comment by MartinPacker** [comment]
Thanks for your kind words. I will investigate what we can do. (I'm assuming I didn't already provide metadata terms for these two - pseudofooter and page number text sizes.)

Question: Does it make any sense for the three pseudofooter sizes to be different?

(And I emphasise **pseudo** as md2pptx does not use the **standard** footer and page number fields. The reason for this is mainly to allow flexibility the standard ones don't allow. I'm also receptive to ideas for further flexibility in the pseudofooters.)

**Comment by MartinPacker** [comment]
So I can confirm this function doesn't yet exist. So tagging as new function.

**Comment by yamsu** [comment]
A single size metadata terms would be sufficient for all three footers and page number. I personally like the fact that the footer are separate from the standard ones.

**Comment by MartinPacker** [comment]
My thinking is 1 for footers - which really ought to be the same size as each other (as you say) - and 1 for slide numbers.

How urgent is this @yamsu?

**Comment by yamsu** [comment]
Take your time, i'm happy to wait.

**Comment by MartinPacker** [comment]
Thank you. The code needs some repair right now - with enhancements that aren't yet ready. None of these changes have been pushed.

By the way, do you clone the repo or just download a release? (Affects how you might test the enhancements you need.)

**Comment by MartinPacker** [comment]
Turns out the code has no regressions that I can see - even if my new features don't work well enough for me to advertise them.

Just pushed

`footerFontSize: 12`

and

`numbersFontSize: 24'

where the `12` and `24` values are floating point and you can obviously change them.

Will write up soon and hope to ship 4.1.1 with these in and documented.

**Comment by MartinPacker** [comment]
Documented and released as [v4.1.1](https://github.com/MartinPacker/md2pptx/releases/tag/v4.1.1).

Closing. Please reopen or, preferably, create another issue if you want this tweaked / extended.

**Comment by yamsu** [comment]
Nice! Thank you
